lib/gyp/bindings.js

```diff
@@ -42,3 +43,15 @@ exports.log = function log(message) {
 exports.error = function error(message) {
   process.stderr.write(message + '\n');
 };
+
+Object.defineProperty(exports, 'win', {get() {
```

Comment:
Let's be a bit more verbose: `get: function get() {`. Also, please put it on separate line and indent properly.